File: Python/824.py
class Solution(object):
    def toGoatLatin(self, sentence):
        vowel = "aeiouAEIOU"
        sentence = sentence.split()
        word = ""
        for i in range(len(sentence)):
            if sentence[i][0] in vowel:
                word += sentence[i]
            else:
                word += sentence[i][1:] + sentence[i][0]
            word += "ma" + "a"*(i+1)
            if i != len(sentence)-1:
                word += " "
        return word

File: Java/824.java
import java.util.*;
class Solution {
    public String toGoatLatin(String sentence) {
        String vowel = "aeiouAEIOU", word = "",a = "";
        ArrayList<String> lst = new ArrayList<>();
        int k = 0,flag = 0;
        for(int i = 0; i < sentence.length(); i++){
            if(sentence.charAt(i) == ' '){
                lst.add(sentence.substring(k,i));
                k = i+1;
            }
        }
        lst.add(sentence.substring(k));
        for(int i = 0; i < lst.size(); i++){
            flag = 1;
            for(int j = 0; j < 10; j++){
                if(lst.get(i).charAt(0) == vowel.charAt(j)){
                    flag = 0;
                    break;
                }
            }
            word += lst.get(i).substring(flag) + lst.get(i).substring(0,flag) + "ma";
            a += "a";
            word += a;
            if(i!=lst.size()-1) word += " ";
        }
        return word;
    }
}